samples/apps/book-creator-webapp-react/README.md

```diff
@@ -9,14 +9,16 @@
 
 ## Running the sample
 
-1. You will need an [Open AI Key](https://openai.com/api/) or
+1. Create a new copy of `.env.development` and save it as `.env` next to the original. 
```

Comment:
If they plan to use this locally, they may be ok just leaving it as .env.development (see https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used)

Reply:
Another approach I was thinking is to leave the `.env` files as is while **keeping then ignored**. 

This way I can remove the additional readme step and if we add any other react sample to the repo we can use `git add --force` mechanism to add an empty `.env` template.

i.e.:
`git add --force samples/apps/new-sample/.env`

Reply:
Changes made!